### PR TITLE
EDU-1351 Allow comments on document inputs

### PR DESCRIPTION
**Pull request by adrianaluchian**
Closes https://educandu.atlassian.net/browse/EDU-1351

### CODE REVIEW

File: src/components/room/room-document-inputs.js [hunk 1]
```diff
@@ -28,6 +28,9 @@ export default function RoomDocumentInputs({ documentInputs, onDelete }) {
   const renderDocumentInput = documentInput => {
     const url = routes.getDocumentInputUrl(documentInput._id);
     const userProfileUrl = routes.getUserProfileUrl(documentInput.createdBy._id);
+    const commentsCount = Object.values(documentInput.sections)
+      .map(sectionData => sectionData.comments)
+      .flat().length;
```
Comment (ahelmberger) [inline src/components/room/room-document-inputs.js:33]: Same here

File: src/components/section-display.js [hunk 2]
```diff
@@ -215,13 +217,22 @@ function SectionDisplay({
...
-        />
+      <Fragment>
+        <DisplayComponent
+          context={context}
+          content={section.content}
+          input={sectionInput?.data ?? null}
+          canModifyInput={canModifyInput}
+          onInputChanged={onSectionInputChange}
+          />
+        {!!registeredPlugin.info.allowsInput && !!documentInputId && (
```
Comment (ahelmberger) [inline src/components/section-display.js:228]: Maybe also for deleted sections: `&& !section.deletedOn`?

Comment (ahelmberger) [reply src/components/section-display.js:228]: Although - that might hide historical comments :thinking:

Comment (ahelmberger) [reply src/components/section-display.js:228]: So I think we should keep it as is, please ignore all the comments on this line :smile:

File: src/components/dashboard/document-inputs-tab.js [hunk 3]
```diff
@@ -46,6 +46,9 @@ function DocumentInputsTab({ loading, documentInputs, onDeleteDocumentInput }) {
           <div>
             {documentInputs.map(documentInput => {
               const url = routes.getDocumentInputUrl(documentInput._id);
+              const commentsCount = Object.values(documentInput.sections)
+                .map(sectionData => sectionData.comments)
+                .flat().length;
```
[the remark this answers is not in the document]
Comment (adrianaluchian) [reply src/components/dashboard/document-inputs-tab.js]: Nice, thanks, I've used this now.

File: src/components/section-display.js (hunk 2 above)
Comment (adrianaluchian) [reply src/components/section-display.js:228]: What comments? 😵‍💫

**Review by ahelmberger — APPROVED** [review]

**Pull request merged**